docs/.custom_wordlist.txt

```diff
@@ -32,3 +32,7 @@ snapcraft
 Snapcraft
 toolkits
 hostpath
+Validator
+CUDA
+DaemonSet
+utilize
```

Comment:
Wondering why we add this one - shouldn't we use UK style? I was checking the style guide but couldn't find an exception for "utilize".

Reply:
Agree, removed this from custom words